docs/install/setting_up_cluster_on_amazon_web_services.md

```diff
@@ -208,31 +205,35 @@ Here:
 
 * `<your-keyfile-path>` refers to the path to the key file.
 
-* `<action>` refers to the action to be performed. You must first launch your cluster using the `launch` action. The `start` and `stop` actions can be used to manage the nodes in the cluster.
+* `<action>` refers to the action to be performed. Some of the available actions are `launch`, `destroy`, `stop`, `start` and `reboot-cluster`.
+Use `launch` action to create a new cluster while `stop` and `start` actions work on existing clusters.
 
 By default, the script starts one instance of a locator, lead, and server each.
-The script identifies each cluster by its unique cluster name (you provided) and internally ties members (locators, leads, and stores/servers) of the cluster with EC2 security groups.
+The script identifies each cluster by its unique cluster name (that you provide) and internally ties members (locators, leads, and stores/servers) of the cluster with EC2 security groups - whose names are also derived from the cluster name.
 
-The  names and details of the members are automatically derived from the provided cluster name. When running the script you can also specify properties like the number of stores and region.
+When running the script you can also specify options like the number of stores in the cluster and region where the EC2 instances should be launched.
 
 **Example**
 
 ```pre
 ./snappy-ec2 -k my-ec2-key -i ~/my-ec2-key.pem --stores=2 --with-zeppelin --region=us-west-1 launch my-cluster
 ```
 
-In the above example, you are launching a SnappyData cluster named **my-cluster** with 2 stores (or servers). The locator is available in security group named **my-cluster-locator** and the store/server are available in **my-cluster-store**.
+The above example will launch a SnappyData cluster named **my-cluster** with 2 stores (or servers). The locator will be associated with security group named **my-cluster-locator** and the servers will be associated with **my-cluster-store** security group.
+
+The cluster will be launched in the N. California (us-west-1) region on AWS and it'll also have an Apache Zeppelin server running on the instance where the lead is running.</br>
 
-The cluster is launched in the N. California (us-west-1) region on AWS and starts an Apache Zeppelin server on the instance where the lead is running.</br>
 The example assumes that you have the key file (my-ec2-key.pem) in your home directory for EC2 Key Pair named 'my-ec2-key'.
 
 !!! Note
-	By default, the cluster is launched in the US East (N. Virginia) region on AWS. To launch the cluster in a specific region ensure that you set the region property `--region=`.
+	By default, the cluster is launched in the US East (N. Virginia) region on AWS. To launch the cluster in a specific region use option `--region`.
```

Comment:
By default, the cluster is launched in the **US East (N. Virginia)** region on AWS. To launch the cluster in a specific region use option `--region`.